reconcile/queries.py

```diff
@@ -102,6 +102,11 @@ def get_credentials_requests():
     return gqlapi.query(CREDENTIALS_REQUESTS_QUERY)['credentials_requests']
 
 
+def get_integrations():
+    gqlapi = gql.get_api()
+    return gqlapi.query(gql.INTEGRATIONS_QUERY)['integrations']
```

Comment:
an interesting exception. i can't think of anything better to do here.

Reply:
yes, I had it the other way round, but it created a circular dependency